adapters/taboola/taboola.go

```diff
@@ -176,8 +173,14 @@ func createTaboolaRequests(request *openrtb2.BidRequest) (taboolaRequests []*ope
 			errs = append(errs, err)
 			continue
 		}
-		if taboolaExt.TagId != "" {
-			imp.TagID = taboolaExt.TagId
+
+		tagId := taboolaExt.TagID
+		if len(taboolaExt.TagID) < 1 {
+			tagId = taboolaExt.TagId
+		}
+
+		if tagId != "" {
```

Comment:
[static/bidder-params/taboola.json](https://github.com/prebid/prebid-server/pull/2662/files?diff=split&w=0#diff-2e3ad40931665cd4a88603fa81950e65c1c86e2cacd0416166d5b2fd1fec48a7) mentions that any one of `tagid` or `tagId` is required.  
Therefore `tagId != ""` is not needed. Core functionality should handle this.